src/main/resources/assets/tconstruct/book/puny_smelting/en_us/upgrades/ranged/impaling.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:impaling",
+  "text": [
+    {
+      "text": "I'd be rather appalled if you impaled me with impaling."
```

Comment:
Lets add a line here, "Not to be confused with anti-aquatic" for the vanilla people. May be worth adding a similar line to the piercing page, "Not to be confused with impaling"